bin/nimbus-fml.sh

```diff
@@ -164,14 +164,22 @@ if [ -n "$MOZ_APPSERVICES_LOCAL" ] ; then
     LOCAL_FML_DIR="$MOZ_APPSERVICES_LOCAL/components/support/nimbus-fml"
     export BINARY_PATH="$HOME/.cargo/bin/cargo run --manifest-path $LOCAL_FML_DIR/Cargo.toml --"
 else
-    # Otherwise, we should download a pre-built copy from github.com/mozilla/application-services/releases
+    # Otherwise, we should download a pre-built copy
+    AS_DOWNLOAD_URL="https://github.com/mozilla/application-services/releases/download/v$AS_VERSION"
```

Comment:
@bendk We should file a ticket to remember to switch this if we move the artifacts out of github releases for the 115 release